### DISCUSSION TITLE
How to import and export data layer of geoman map?

### DISCUSSION THREAD

**Discussion by hamish0361** [geoman-io/leaflet-geoman#1182] [Questions & Help]
Can you show me how to import and export the data layer of the map?
Each of my layers will have properties like color or Text note for each marker to store in data via API and when getting data I can pass in map leaflet-geoman and make edits to it.
Thanks very much and have a nice working day.

**Comment by Falke-Design** [comment]
Please look into #856,  #858

**Reply by hamish0361** [reply]
Thank you for your answer I also had a look at the functions you wrote about importing and exporting data but for Text Layer I can't do export and import with text with location and its information.
So can you show me the function import and export TextLayer in geomap.
Thank you so much.
